ee/packages/omni-lib-ee/src/index.ts

```diff
@@ -0,0 +1,5 @@
+import { isDepartmentCreationAvailablePatch } from './isDepartmentCreationAvailable';
+
+export function patchOmniLib(): void {
```

Comment:
Should we expose a function to unpatch as well?

Reply:
I don't see that feature in the `makeFunction`:

https://github.com/RocketChat/Rocket.Chat/blob/abe83209363e8ea4336afde0c8d6801d2526b5b2/packages/patch-injection/src/makeFunction.ts#L5

Reply:
The `patch` one has it
```
export const addPatch = <T extends BaseFunction>(baseFunction: T, patch: PatchFunction<T>, condition?: () => boolean) => {
	const patches = getFunctionPatches(baseFunction);

	const patchData: PatchData<T> = {
		patchFunction: patch,
		condition,
	};

	patches.add(patchData);

	return () => {
		patches.delete(patchData);
	};
};
```